src/main/native/jgskit.mac.mak

```diff
@@ -24,56 +21,67 @@ JAVACLASSDIR=${TOPDIR}/target/classes
 #This flag must be disabled before building production version
 #DEBUG_DATA =  -DDEBUG_DH_DATA -DDEBUG_DSA_DATA -DDEBUG_EC_DATA -DDEBUG_GCM_DATA -DDEBUG_CCM_DATA -DDEBUG_HMAC_DATA -DDEBUG_CIPHER_DATA -DDEBUG_RSA_DATA -DDEBUG_SIGNATURE_DATA -DDEBUG_SIGNATURE_DSANONE_DATA -DDEBUG_SIGNATURE_RSASSL_DATA -DDEBUG_HKDF_DATA -DDEBUG_RSAPSS_DATA -DDEBUG_SIGNATURE_EDDSA_DATA
 #DEBUG_FLAGS+= -g ${DEBUG_DETAIL}  ${DEBUG_DATA}
```

Comment:
Is there a reason to remove the debug flags from here? If we ever need them it will be a pain to get them to add in.

Reply:
I didn't remove them. They have just been moved around to make all files look similar. It's just the diff that makes it look like they've been totally removed.